components/multiselect/multiselect.jsx

```diff
@@ -67,7 +67,7 @@ export default class MultiSelect extends React.Component {
         }
 
         for (let i = 0; i < this.props.values.length; i++) {
-            if (this.props.values[i].value === value.value) {
+            if (this.props.values[i].username === value.username) {
```

Comment:
This component should be generic and not have any references to users, channels, teams, and so on. Not sure what this is fixing, but can you do it in a generic way?

Reply:
Ok, I think the solution might be to wrap all these objects so that there's something similar to:

```
{
  id: "someidentifer", // could be username, team name, user id etc.
  value: theactualvalue // an object holding the actual value
}
```

Know what I'm getting at?

Reply:
Actually, I did some digging and noticed that the `value` object being passed in both have an `id` field, so I think it's ok to just use that as it's also generic.